src/BizHawk.Client.DiscoHawk/BizHawk.Client.DiscoHawk.csproj

```diff
@@ -70,13 +70,13 @@
 		<Content Include="discohawk.ico" />
 	</ItemGroup>
 	<ItemGroup>
-		<Compile Update="About.cs" SubType="Form" />

```

Comment:
Did you mean to change this file?

Reply:
Nope, this must've been VS. I reverted it back to the previous state.